Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,18 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const totalEarnings = tasks
+    .map(x => x.duration / 60)
+    .filter(x => x >= 2)
+    .map(x => x * hourlyRate)
+    .reduce((accumulator, number) => accumulator + number);
+  return totalEarnings;
 }
```

Comment:
You could write the function like this, to show through the proper naming of parameters how the data is transformed and filter while it travels through the chain, without requiring the reader to do mental mapping:

```js
function computeEarnings(tasks, hourlyRate) {
  return tasks
    .map(task => task.duration / 60)
    .filter(hour => hour >= 2)
    .map(hour => hour * hourlyRate)
    .reduce((total, amount) => total + amount, 0);
}
```